docs/gx_time_frequency.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: Time-Frequency component
+tagline: GreenX Time-Frequency
+description: Time-Frequency component
+---
+
+# Description
+
+# Usage
+
+# Benchmarks
+
+## CH<sub>4</sub> RPA
+
+In this test, we evaluate the RPA total energy of CH4 using a Gauss-Legendre grid, a modified Gauss-Legendre grid (so far the standard in FHI-aims and abinit), and minimax grids. An accuracy of 10^-6 eV is reached with 10 minimax grid points while the modified Gauss-Legendre grids requires 36 points for this accuracy.
+
+[CH4 benchmark](./img/ch4_becnh.png)
```

Comment:
`ch4_becnh.png` should be renamed `ch4_bench.png`

Reply:
Done